### DISCUSSION TITLE
安装  AdGuard Home 遇到端口问题

### DISCUSSION THREAD

**Discussion by Mask-Vip** [1Panel-dev/1Panel#5131] [Q&A]
### 1Panel 版本

v1.10.8-lts 社区版 UbntuServer22最新版

### 请描述您的需求或者改进建议

各位大佬,小白面板战士又来求教了.

今天尝试安装应用商店里的AdGuard Home应用,由于是初次安装,一开始一直无法导入流量,后来差看了半天教程才发现.1panel商店里的AdGuard Home在安装的时候,默认端口竟然全都改了,尤其是nds的默认端口53改成了20053.这就导致了,在爱快软路由里设置dns指向到AdGuard Home服务器ip后也无法连接dns服务,而且爱快软路由还不支持dns地址带端口.然后就卡到这里了.

然后卸载AdGuard Home想着重新安装的时候再把20053端口改成dns默认端口53,结果在安装的时候又提示53端口已被占用,无法完成安装,这第二个方法也就卡到这里了.

现在不知道如何是好了,求教各位大佬,看如何能默认53端口成功安装AdGuard Home呀?应为改成20053非默认端口的话,有很多地方还是无法兼容或是需要每台设备一个一个去设置.

**Comment by ltxhhz** [comment]
你可以参考 #5827 
先查看占用端口的进程，然后选择关闭或者监听其他ip
我选择了监听其他ip

**Comment by nanyancc** [comment]
I used `lsof` and found the following:

```shell
root@Ali:~/compose/AdGuardHome# lsof -i :53
COMMAND   PID            USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
systemd-r 691 systemd-resolve   13u  IPv4  19854      0t0  UDP localhost:domain 
systemd-r 691 systemd-resolve   14u  IPv4  19855  
```

Since I no longer need `systemd-resolve`, I decided to disable it:

```shell
sudo systemctl stop systemd-resolved
sudo systemctl disable systemd-resolved
```

However, I encountered another issue. The client at `172.18.0.1` keeps querying `o1098464.ingest.sentry.io`, which is unrelated to my server. This is quite strange, and I am unable to add another domain or create a new website because it times out.


<img width="1193" alt="image" src="https://github.com/user-attachments/assets/42ca1b9c-af14-4a91-9813-3e4e4467862b">
